nodeup/pkg/model/protokube.go

```diff
@@ -108,12 +108,27 @@ func (t *ProtokubeBuilder) buildSystemdService() (*nodetasks.Service, error) {
 		"-v", "/:/rootfs/",
 		"-v", "/var/run/dbus:/var/run/dbus",
 		"-v", "/run/systemd:/run/systemd",
-		"--net=host", "--privileged",
+	}
+
+	// add kubectl only if a master
+	// path changes depending on distro, and always mount it on /opt/kops/bin
+	// kubectl is downloaded an installed by other tasks
+	if t.IsMaster {
+		dockerArgs = append(dockerArgs, []string{
+			"-v", t.KubectlPath() + ":/opt/kops/bin:ro",
+			"--env", "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/kops/bin",
```

Comment:
Nit: Probably better to put `/opt/kops/bin` at the start, except that we know that most of these dirs don't exist because we're in a container...

I guess the problem here is that `channels` calls `kubectl`, so we have to make sure it's on the path?  If that's the case, can we have a comment, as otherwise the temptation will be to just change `kubectl` calls to be a fully-qualified path.

Reply:
Well ... I disagree.  We need it to be at the end, so that other stuff does not get whacked.  First, wins in bash?

Reply:
OK - we don't expect many copies of kubectl :-)

Can we have a comment though about why we don't just call kubectl with an absolute path, i.e. why we are messing with the PATH at all

Reply:
Cause we do not want to make changes to channels :P  Yes I will make a comment.